data/net/minecraft/util/Mth.mapping

```diff
@@ -89,13 +106,32 @@ CLASS net/minecraft/util/Mth
 		ARG 0 r
 		ARG 1 g
 		ARG 2 b
+	METHOD colorMultiply (IFFF)I
+		COMMENT Multiplies an RGB color with the three floats for red, green and blue that should be in range [0;1].
```

Comment:
Perhaps it would be better if the range of values of the floats were documented on the parameters themselves?